_studio/mfx_lib/decode/vp9/include/mfx_vp9_dec_decode_hw.h

```diff
@@ -124,6 +124,10 @@ class VideoDECODEVP9_HW : public VideoDECODE, public MfxCriticalErrorHandler
     typedef std::list<mfxFrameSurface1 *> StatuReportList;
     StatuReportList m_completedList;
 
+    std::vector<UMC::FrameMemID> m_bufferedSurfaces;
```

Comment:
We have [m_frameInfo.ref_frame_map], why do we need yet another array of IDs?
Anyway, even If we really need something like that, u have to sync w/ [PR #1114](https://github.com/Intel-Media-SDK/MediaSDK/pull/1114) which introduces _ReferenceFrameStorage_ that in turn has _m_lockedRefFrames_ map & _m_decodedFramesIDs_ array.